### ISSUE TITLE
StringTie assigned same gene_id to non-overlapped genes

### ISSUE THREAD

**Issue by kepbod** [gpertea/stringtie#116]
I found that for some genes StringTie assigned same gene_id to non-overlapped genes as shown below. It is very wired. How could I let StringTie assign different gene_id because the gene_id information is very import for downstream analysis? Thank.

```
chr20	StringTie	transcript	35405845	35492089	1000	-	.	gene_id "STRG.1516"; transcript_id "STRG.1516.1"; reference_id "ENST00000237536.4"; ref_gene_id "ENSG00000149639.10"; ref_gene_name "SOGA1"; cov "14.863372"; FPKM "45.463112"; TPM "150.084625";
chr20	StringTie	transcript	35414888	35488276	1000	-	.	gene_id "STRG.1516"; transcript_id "STRG.1516.2"; reference_id "ENST00000357779.3"; ref_gene_id "ENSG00000149639.10"; ref_gene_name "SOGA1"; cov "0.713037"; FPKM "2.180991"; TPM "7.199974";
chr20	StringTie	transcript	35457000	35467805	1000	-	.	gene_id "STRG.1516"; transcript_id "STRG.1516.3"; reference_id "ENST00000463491.1"; ref_gene_id "ENSG00000149639.10"; ref_gene_name "SOGA1"; cov "0.596945"; FPKM "1.825896"; TPM "6.027722";
chr20	StringTie	transcript	35405845	35492089	1000	-	.	gene_id "STRG.1516"; transcript_id "STRG.1516.4"; reference_id "ENSG00000149639.10"; ref_gene_id "ENSG00000149639.10"; ref_gene_name "SOGA1"; cov "1.928192"; FPKM "5.897828"; TPM "19.470144";
chr20	StringTie	transcript	35624752	35724398	1000	-	.	gene_id "STRG.1516"; transcript_id "STRG.1516.5"; reference_id "ENST00000373664.3"; ref_gene_id "ENSG00000080839.7"; ref_gene_name "RBL1"; cov "107.594803"; FPKM "329.103943"; TPM "1086.451050";
chr20	StringTie	transcript	35631977	35724392	1000	-	.	gene_id "STRG.1516"; transcript_id "STRG.1516.6"; reference_id "ENST00000344359.3"; ref_gene_id "ENSG00000080839.7"; ref_gene_name "RBL1"; cov "0.619260"; FPKM "1.894152"; TPM "6.253050";
chr20	StringTie	transcript	35624752	35724392	1000	-	.	gene_id "STRG.1516"; transcript_id "STRG.1516.7"; cov "11.723077"; FPKM "35.857780"; TPM "118.375137";
chr20	StringTie	transcript	35624752	35724392	1000	-	.	gene_id "STRG.1516"; transcript_id "STRG.1516.8"; cov "4.109442"; FPKM "12.569694"; TPM "41.495571";
chr20	StringTie	transcript	35651321	35724392	1000	-	.	gene_id "STRG.1516"; transcript_id "STRG.1516.9"; cov "2.706278"; FPKM "8.277786"; TPM "27.326958";
chr20	StringTie	transcript	35696490	35724398	1000	-	.	gene_id "STRG.1516"; transcript_id "STRG.1516.10"; reference_id "ENST00000527999.1"; ref_gene_id "ENSG00000080839.7"; ref_gene_name "RBL1"; cov "0.002006"; FPKM "0.006136"; TPM "0.020256";
chr20	StringTie	transcript	35650146	35650387	1000	-	.	gene_id "STRG.1516"; transcript_id "STRG.1516.11"; reference_id "ENST00000411933.1"; ref_gene_id "ENSG00000231241.1"; ref_gene_name "RPS3AP3"; cov "0.163259"; FPKM "0.499366"; TPM "1.648527";
chr20	StringTie	transcript	35650146	35650387	1000	-	.	gene_id "STRG.1516"; transcript_id "STRG.1516.12"; reference_id "ENSG00000231241.1"; ref_gene_id "ENSG00000231241.1"; ref_gene_name "RPS3AP3"; cov "0.160395"; FPKM "0.490606"; TPM "1.619607";
chr20	StringTie	transcript	35624752	35724398	1000	-	.	gene_id "STRG.1516"; transcript_id "STRG.1516.13"; reference_id "ENSG00000080839.7"; ref_gene_id "ENSG00000080839.7"; ref_gene_name "RBL1"; cov "0.000474"; FPKM "0.001450"; TPM "0.004786";
```

**Comment by kepbod** [comment]
This bug has been fixed in 1.3.3b. Thanks so much.

**Comment by JFourquet** [comment]
I have the same issue with the 1.3.3b version of Stringtie. Do you know how can I fixed it ?
